metricbeat/module/vsphere/resourcepool/_meta/fields.yml

```diff
@@ -0,0 +1,85 @@
+- name: resourcepool
+  type: group
+  release: beta
+  description: >
+    resourcepool
+  fields:
+    - name: cpu
+      type: group
+      fields:
+        - name: entitlement
+          type: group
+          fields:
+            - name: mhz
+              type: long
+              description: >
+                Amount of CPU resources allocated to the resource pool.
+        - name: usage
+          type: group
+          fields:
+            - name: pct
+              type: long
+              description: >
+                CPU usage average in percentage.
+            - name: mhz
+              type: long
+              description: >
+                CPU usage average in MHz.
+        - name: active
+          type: group
+          fields:
+            - name: average
+              type: group
+              fields:
+                - name: pct
+                  type: long
+                  description: >
+                    CPU active average in percentage.
+        - name: max
```

Comment:
This should be the child of `active` as per [data.json](https://github.com/elastic/beats/pull/40456/files#diff-38cd9db279dab0838e20f23a4dbbfbdd6e5f12501f80d6d3e21cba1a9393a69cR21-R28) file and the field description!